WPAppointmentManager/main.py

```diff
@@ -0,0 +1,89 @@
+import logging
+import pprint
+from datetime import datetime, timedelta
+from couchdb_client import CouchDbClient
+from wp_client import WordpressClient
+from mail_client import MailClient
+
+logging.basicConfig(level=logging.DEBUG)
+logging.getLogger().setLevel(logging.DEBUG)
+couchdb_client = CouchDbClient()
+
+
+def all_items_instock(item_ids):
+    item_status = couchdb_client.get_status_of_items(item_ids)
+    item_ids_found_in_db = list(map(lambda status: status["id"], item_status))
+    item_ids_not_found_in_db = list(set(item_ids) - set(item_ids_found_in_db))
+    if len(item_ids_not_found_in_db) > 0:
+        logging.info(f"Could not find items in db: {item_ids_not_found_in_db}")
+        return False
+    for status in item_status:
+        if status["status"] != "instock":
+            logging.debug(f"item {status['id']} is {status['status']}")
+            return False
+    return True
+
+
+def reserve_items(items):
+    for item_doc in couchdb_client.get_items(items):
+        item_doc = couchdb_client.as_document(item_doc)
+        if not item_doc["exists_more_than_once"]:
+            item_doc["status"] = "reserved"
+            item_doc.save()
+            logging.debug(f"Reserved item {item_doc['id']}")
+        else:
+            logging.debug(f"Did not reserve item {item_doc['id']} because it exists more than once")
+
+
+def appointment_to_string(appointment):
+    return f"appointment at {appointment['time_start'].strftime('%d.%m %H:%M')} with id {appointment['appointment_id']}"
+
+
+def should_auto_accept(appointment):
+
+    if appointment['status']=='Bitte ansehen':
```

Comment:
This can be deleted, because all appointments with status != „Pending“ are filtered in line 66.